content/topics/web-frontend/intro-to-react/_index.md

```diff
@@ -1,3 +1,6 @@
 ---
 title: Intro to React
+
 ---
+
+- [Introduction to the React](https://www.freecodecamp.org/learn/front-end-libraries/react/) - React, created by Facebook, is an open-source JavaScript library for building user interfaces. It is used to create components, handle state and props, utilize event listeners and certain life cycle methods to update data as it changes.
```

Comment:
How about them official docs? The official docs are up to date and very thorough, it's probably worth a mention.

Link to the official tutorial and other stuff.